### ISSUE TITLE
Import profile bugged WeakAura

### ISSUE THREAD

**Issue by pedrotadini** [WeakAuras/WeakAuras2#4725]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues.

### Description

I'm playing 3.3.5 WotLK, tried to import https://wago.io/En4ShAUoH and the screen got stuck to the Import loading bar. Now when I click my WeakAura icon on the minimap, it doesnt open. I would like to remove the profile that I try to import. I catch this log with Swatter, hope it helps.

```
Date: 2023-11-22 11:59:00
ID: 2
Error occured in: Global
Count: 1
Message: ..\AddOns\WeakAuras\WeakAuras.lua line 650:
   bad argument #1 to 'format' (string expected, got table)
[C]: format'
..\AddOns\WeakAuras\WeakAuras.lua line 650:
   in function <..\AddOns\WeakAuras\WeakAuras.lua:540>
..\AddOns\WeakAuras\WeakAuras.lua line 2427:
   in function <..\AddOns\WeakAuras\WeakAuras.lua:2356>
..\AddOns\WeakAuras\WeakAuras.lua line 2502:
   in function Add'
..\AddOns\WeakAuras\WeakAuras.lua line 1992:
   in function `load'
..\AddOns\WeakAuras\WeakAuras.lua line 1999:
   in function `AddMany'
..\AddOns\WeakAuras\WeakAuras.lua line 969:
   in function <..\AddOns\WeakAuras\WeakAuras.lua:941>

Debug:
   WeakAuras\WeakAuras.lua:1007: Login()
   WeakAuras\WeakAuras.lua:1071:
      WeakAuras\WeakAuras.lua:1023
```
[WeakAuras.zip](https://github.com/WeakAuras/WeakAuras2/files/13441218/WeakAuras.zip)

### WeakAuras Version

4.0.0

### World of Warcraft Flavor

Wrath of the Lich King Classic

### World of Warcraft Region

US/NA

### Tested with only WeakAuras

- [X] Yes

### Reproduction Steps

Go to https://wago.io/En4ShAUoH
Click copy import string
Open WeakAura
Import string

**Comment by pedrotadini** [comment]
I'm sorry, didn't know that. I'm playing on private servers. If I deleted WeakAura.lua from SavedVariables would it help? I don't mind setting all my auras again. 

Just trying to figure this out, sorry for bothering.

**Comment by samuelfmartin** [comment]
Hello, I would like to share that I had the same problem, but the suggested solution did not work for me. What I did to solve it was to clear the WTF and Addons folders and stop using the icon I had created to run the game. After that, I reinstalled my Addons and started opening the game using the executable in the root path.

**I can't confirm whether this makes sense, but it was what solved it for me...**
